test/browser/karma.config.cjs

```diff
@@ -60,6 +60,9 @@ module.exports = function (karma) {
     ],
     proxies: {
       '/spec/': '/base/spec/',
+      '/wms': '/base/spec/ol/data/blank.png',
+      '/ImageServer/exportImage': '/base/spec/ol/data/blank.png',
+      '/MapServer/export': '/base/spec/ol/data/blank.png',
```

Comment:
This configuration, in combination with the new `blank.png`, makes it easy to test the new loader with valid local urls and real image loading sequences.